common/transport/amqp/src/amqp.ts

```diff
@@ -34,7 +35,9 @@ export type GenericAmqpBaseCallback<T> = (err: Error | null, result?: T) => void
  * @param   {String}    sdkVersionString        String identifying the type and version of the SDK used.
  */
 export class Amqp {
-  private _amqp: amqp10.Client;
+  private _amqpContainer: any;
+  private _amqpConnection: any;
```

Comment:
these could now be strongly typed if Amar's contribution to Rhea has been checked in and released